app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+
+def calculate_team_total_rating(team: list) -> int:
+    result = 0
+    for player in team:
+        result += player.get_rating()
+    return result
+
+
+def elves_concert(elves: list) -> None:
```

Comment:
provide full type annotations for elves: list[Elf]
The same for feast_of_the_dwarves